CMakeLists.txt

```diff
@@ -92,6 +92,12 @@ option(disable_handle_fork "Prohibit installation of pthread_atfork() handlers"
 option(install_headers "Install header and pkg-config metadata files" ON)
 option(with_libatomic_ops "Use an external libatomic_ops" OFF)
 option(without_libatomic_ops "Use atomic_ops.h in libatomic_ops/src" OFF)
+option(emscripten_asyncify "Use Emscripten's asyncify feature. Enable this if your program is targeting -sASYNCIFY, and disable otherwise" OFF)
```

Comment:
Please delete the change in the build script, the client could pass -DCFLAGS_EXTRA="-DEMSCRIPTEN_ASYNCIFY" instead

Reply:
Okay, let me think first more about it - I will reply in a day or two

Reply:
I have renamed the cmake option to enable_emscripten_asyncify